vendor/github.com/keybase/client/go/protocol/keybase1/extras.go

```diff
@@ -1664,6 +1664,7 @@ func (u UserPlusKeysV2) ToUserVersion() UserVersion {
 	}
 }
 
+// Can return nil.
```

Comment:
Perhaps name it `GetLatestPerUserKeyIfPossible()` to suggest this in name directly?